grayskull/strategy/pypi.py

```diff
@@ -401,11 +401,36 @@ def get_metadata(recipe, config) -> dict:
     test_requirements = optional_requirements.pop(config.extras_require_test, [])
     test_section = compose_test_section(metadata, test_requirements)
 
+    # Compute home, doc_url, and dev_url for the "about" section
+
+    if metadata.get("project_urls") and metadata["project_urls"].get("Homepage"):
```

Comment:
can you encapsulate all this handling for the URLs in a function please?
so we can test that easily :)